browser-sign-in/__tests__/App-test.js

```diff
@@ -193,11 +188,10 @@ describe('when authenticated', () => {
     wrapper.setState({authenticated: true});
     const profileButton = wrapper.find('Button').get(3);
     await profileButton.props.onPress();
-    setTimeout( function() { 
-      expect.assertions(2);
-      expect(profileButton.props.title).toBe('Get User From Access Token');
-      expect(wrapper.state().context).toContain('foo');
-    }, 100);
+    await waitForState(wrapper, state => state.context !== null);
+    expect.assertions(2);
```

Comment:
Is this assertions() call required?  Since you're not in a callback it should be able to automatically see the number of expect()s, meaning this code is just a chance to forget to update in the future.

Reply:
I deleted these assertions checks, I think since it is using await so each expect is one assertion check